lib/todo.js

```diff
@@ -16,7 +24,7 @@ const Todo = {
       title: 'RegExp Pattern',
       description: 'used in conjunction with RegExp Flags to find todo items in your code',
       type: 'string',
-      default: 'TODO\\:.+',
+      default: '(TODO|FIXME|CHANGED|XXX|IDEA|HACK|NOTE|REVIEW|NB|BUG|QUESTION|COMBAK|TEMP)\\:.+',
```

Comment:
I think this part is really cool and could fit in its own small PR for #33 while we figure out the rest.

Reply:
since the change is very small maybe you could just modify it using the github editor